pkg/asset/templates/kube-apiserver.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     k8s-app: kube-apiserver
     version: v1.3.6_coreos.0
```

Comment:
I know you wanted to do this before: until we handle updating label (which I'd rather not bother with right now), we should probably remove these (otherwise image version won't match label anymore)